recipes/rcorrector/build.sh

```diff
@@ -5,7 +5,7 @@ export OBJC_INCLUDE_PATH=$PREFIX/include
 export CPLUS_INCLUDE_PATH=$PREFIX/include 
 export LD_LIBRARY_PATH=$PATH/lib:$LD_LIBRARY_PATH
 
-
 make
 mkdir -p $PREFIX/bin
-cp -R rcorrector run_rcorrector.pl $PREFIX/bin
\ No newline at end of file
+cp -R rcorrector run_rcorrector.pl $PREFIX/bin
+chmod -x $PREFIX/bin/run_rcorrector.pl
```

Comment:
@bgruening Did you mean `chmod +x` ?

Reply:
Should be fixed by https://github.com/bioconda/bioconda-recipes/pull/10794 .

Reply:
Yes, most likely :(